subsys/dfu/src/dfu_target_modem.c

```diff
@@ -53,6 +53,34 @@ static int apply_modem_upgrade(void)
 	}
 	return 0;
 }
+
+static int open_modem_dfu_socket(void)
+{
+	fd_dfu = socket(AF_LOCAL, SOCK_STREAM, NPROTO_DFU);
+	return fd_dfu;
+}
+
+static int close_modem_dfu_socket(void)
+{
+	return close(fd_dfu);
+}
```

Comment:
When you create these one-line wrappers, you could add a log line so that you log the errors in one common place and clean up the callers a bit, like you do for `get_modem_offset`.

Reply:
Done.